### ISSUE TITLE
Is there a way to make a part add 2 objects in the metadata?

### ISSUE THREAD

**Issue by VitorAugustoCunha** [HashLips/hashlips_art_engine#1091]
I have a part of a hair and a hat but the hair passes the hat, I would like to know if there is a way I can make a specific design for that hat with the hair, it would just be an image containing 2 parts of the metadata example:

Hair1 + Hat1 = HAIR1andHAT1

**Comment by bolshoytoster** [comment]
@VitorAugustoCunha no, the easiest way to do this would be to have the hair and hats be on the same layer.
